### PR TITLE
v1.11 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
 # Mattermost Mobile Apps Changelog
 
+## v1.11.0 Release
+- Release Date: August 16, 2018
+- Server Versions Supported: Server v4.0+ is required, Self-Signed SSL Certificates are not supported
```
Comment (enahum) [inline CHANGELOG.md]: Lets bump this to the ESR version

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+
+### Highlights
+
+#### Searching Archived Channels
+- Added ability to search for archived channels on desktop and mobile clients.
+
+#### Deep Linking
+-
```
Comment (enahum) [inline CHANGELOG.md]: This needs docs, they should be in the PR but we should include them in the mobile docs, this also needs a custom build.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+
+#### Searching Archived Channels
+- Added ability to search for archived channels on desktop and mobile clients.
+
+#### Deep Linking
+- 
+
+### Improvements
+- Added profile pop-up to combined system messages.
+- Updated the home screen icon for Android.
```
Comment (enahum) [inline CHANGELOG.md]: I don't think the screen icon is an improvement, what could be an improvement is Force re-entering SSO auth credentials after logout

Comment (enahum) [reply CHANGELOG.md]: Improvements on consecutive posts by the same user

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+- Fixed an issue where the previous server URL was present on the input textbox before changing the screen to Login.
+- Fixed an issue where user menu was not translated after App was translated.
+- Fixed an issue where maximum field lengths didn't match web app when editing Account Settings.
+- Fixed an issue where long URLs for embedded images in message attachments got cut off and didn't render.
+- Fixed an issue where link preview images were not cropped properly.
+- Fixed an issue where long usernames didn't wrap properly in the Account Settings menu.
+- Fixed an issue where DMs would not open if users were using Jump To.
+- Fixed an issue where no message was displayed after removing a user from a channel with join/leave messages disabled.
+- Fixed an issue where Android showed the wrong time when timezone setting was disabled .
+
```
Comment (enahum) [inline CHANGELOG.md]: * Fix an issue with Video playback when the filename had spaces
* Fix a crash when playing YouTube Videos
* Fix session expiration notification
* Fix sharing files from Google Drive in Android Share Extension

File: CHANGELOG.md (hunk 3 above)
Comment (saturninoabril) [reply CHANGELOG.md]: (Also maybe an improvement from https://github.com/mattermost/mattermost-mobile/pull/1972)
- Added a loading indicator when user info is still loading on LHS

File: CHANGELOG.md (hunk 2 above)
Comment (amyblais) [reply CHANGELOG.md]: https://github.com/mattermost/docs/pull/2218

**Comment by amyblais** [comment]
@enahum I've made the requested changes ^

**Comment by amyblais** [comment]
@esethna Can you help review this?

**Comment by esethna** [comment]
@enahum please help take another look and merge?

**Pull request merged**